### PR TITLE
feat(HeaderBar): Information dropdown

### PR DESCRIPTION
**Pull request by sgendre**
**What is the problem this PR is trying to solve?**
Information actions are within the dropdown of the Help action.

**What is the chosen solution to this problem?**
Information actions are within their own dropdown.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[x] This PR introduces a breaking change**
An information dropdown is provided.

### CODE REVIEW

File: packages/components/src/HeaderBar/HeaderBar.component.js.orig [hunk 1]
```diff
@@ -0,0 +1,355 @@
+import PropTypes from 'prop-types';
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.component.js.orig]: This file should not be here :)

File: packages/components/src/HeaderBar/HeaderBar.scss [hunk 2]
```diff
@@ -17,6 +17,15 @@ $background-color-on-hover: shade($brand-primary, 30);
 	width: 100%;
 	z-index: 1030;
 
+	& ul li {
+		a svg {
+			margin: 0 $padding-smaller;
+		}
+		div button span svg {
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.scss]: wow! A simple `svg` selector is not enough?

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 3]
```diff
@@ -219,11 +237,14 @@ function HeaderBar(props) {
 			</ul>
 			<ul className={classNames(theme['tc-header-bar-actions'], theme.right)}>
 				{props.search && <Components.Search renderers={renderers} {...props.search} />}
-				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
-				{props.user && <Components.User renderers={renderers} {...props.user} />}
 				{props.notification && (
 					<Components.AppNotification renderers={renderers} {...props.notification} t={props.t} />
 				)}
+				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
+				{!props.user && (
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: What do you think about usging function here instead?

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 4]
```diff
@@ -108,10 +108,27 @@ function Help({ renderers, t, ...props }) {
...
+
+function Information({ renderers, t, ...props }) {
+	const global = {
+		bsStyle: 'link',
+		icon: 'talend-info-circle',
+		label: t('HEADERBAR_INFO', { defaultValue: '' }),
+		tooltipPlacement: 'bottom',
+		...props,
+	};
+	const className = classNames(theme['tc-header-bar-action'], theme.separated);
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: Could you double with plain text className here?
```javascript
const className = classNames(theme['tc-header-bar-action'], 'tc-header-bar-action', theme.separated);
```
To be overrided if needed

File: packages/components/src/HeaderBar/HeaderBar.component.js (hunk 3 above)
Comment (sgendre) [reply packages/components/src/HeaderBar/HeaderBar.component.js]: this seems quite straight forward to me and really quick, concise and easy to read

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 5]
```diff
@@ -108,10 +108,27 @@ function Help({ renderers, t, ...props }) {
...
+			<renderers.Action {...global} />
+		</li>
+	);
+}
+
+function Information({ renderers, t, ...props }) {
+	const global = {
+		bsStyle: 'link',
+		icon: 'talend-info-circle',
+		label: t('HEADERBAR_INFO', { defaultValue: '' }),
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: Could you provide defaultValue for translators please

**Comment by frassinier** [comment]
Looks like a breaking change since props should change to move current help menu under information?

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 6]
```diff
@@ -219,11 +241,14 @@ function HeaderBar(props) {
...
 			<ul className={classNames(theme['tc-header-bar-actions'], theme.right)}>
 				{props.search && <Components.Search renderers={renderers} {...props.search} />}
-				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
-				{props.user && <Components.User renderers={renderers} {...props.user} />}
 				{props.notification && (
 					<Components.AppNotification renderers={renderers} {...props.notification} t={props.t} />
 				)}
+				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
+				{!props.user && (
+					<Components.Information renderers={renderers} {...props.information} t={props.t} />
```
Comment (jmfrancois) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: props.information seems required so you have two choices:
* make it optional (add && props.information)
* add a default value to empty object

Comment (sgendre) [reply packages/components/src/HeaderBar/HeaderBar.component.js]: I don't get your second statement : if I provide an empty object to information, the test will be useless

**Pull request merged**